oauth2-server/src/main/java/com/clouway/oauth2/token/Tokens.java

```diff
@@ -31,9 +30,18 @@
 
   /**
    * Ussues a new token for the provided identity.
+   *
+   * @param client the client to which token will be ussued
    * @param identityId the identityId for which token was issued
    * @param when the requested time on which it should be issued
-   * @return
+   * @return the newly issued token
+   */
+  Token issueToken(Client client, String identityId, DateTime when);
+
+  /**
+   * Revokes token from repository.
+   *
+   * @param token the token which to be revoked
    */
-  Token issueToken(String identityId, DateTime when);
+  void revokeToken(String token);
```

Comment:
Tokens interface is implemented in the applications which are consuming oauth2-server library.